src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Utils/CronExpressionUtilTestsTheoryData.cs

```diff
@@ -102,34 +208,171 @@ public static readonly TheoryData<DateTimeOffset, DateTimeOffset, string, TimeZo
+            "One second before BST starts"
+        },
+        {
+            Dt("2025-03-30T01:00:00 +00:00"),
+            Dt("2025-03-30T09:30:00 +01:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "BST started (expressed in +00:00)"
+        },
+        {
+            Dt("2025-03-30T02:00:00 +01:00"),
+            Dt("2025-03-30T09:30:00 +01:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "BST started (expressed in +01:00)"
+        },
+        {
+            Dt("2025-03-30T01:00:01 +00:00"),
+            Dt("2025-03-30T09:30:00 +01:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "One second after BST started (expressed as +00:00)"
+        },
+        {
+            Dt("2025-03-30T02:00:01 +01:00"),
+            Dt("2025-03-30T09:30:00 +01:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "One second after BST started (expressed as +01:00)"
+        },
+        {
+            Dt("2025-03-30T02:00:00 +00:00"),
+            Dt("2025-03-30T09:30:00 +01:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "One hour after BST started (expressed as +00:00)"
+        },
+        {
+            Dt("2025-03-30T03:00:00 +01:00"),
+            Dt("2025-03-30T09:30:00 +01:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "One hour after BST started (expressed as +01:00)"
+        },
+        {
+            Dt("2025-10-26T00:00:00 +00:00"),
+            Dt("2025-10-26T09:30:00 +00:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "One hour before BST ends (expressed as +00:00)"
+        },
+        {
+            Dt("2025-10-26T01:00:00 +01:00"),
+            Dt("2025-10-26T09:30:00 +00:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "One hour before BST ends (expressed as +01:00)"
+        },
+        {
+            Dt("2025-10-26T00:59:59 +00:00"),
+            Dt("2025-10-26T09:30:00 +00:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "One second before BST ends (expressed as +00:00)"
+        },
+        {
+            Dt("2025-10-26T01:59:59 +01:00"),
+            Dt("2025-10-26T09:30:00 +00:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "One second before BST ends (expressed as +01:00)"
+        },
+        {
+            Dt("2025-10-26T01:00:00 +00:00"),
+            Dt("2025-10-26T09:30:00 +00:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "BST ended"
+        },
+        {
+            Dt("2025-10-26T01:00:01 +00:00"),
+            Dt("2025-10-26T09:30:00 +00:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "One second after BST ended"
+        },
+        {
+            Dt("2025-10-26T02:00:00 +00:00"),
+            Dt("2025-10-26T09:30:00 +00:00"),
+            "30 9 * * *",
+            UkTimeZone,
+            "One hour after BST ended"
+        },
+        // @formatter:on
```

Comment:
I think we should avoid this, because it has reduced the readability of the test cases.
Previously it was easy to compare the times, expression and significance of each case, row by row.
`// @formatter:off` and `// @formatter:on` markers were added deliberately to prevent reformatting.

Replacing them with 

```csharp
// csharpier-ignore-start
// csharpier-ignore-end
```

should prevent it.

This comment also applies to another block of code in this file, and also to one block of code in `PublicationServiceTestsTheoryData`.